invenio_app_ils/config.py

```diff
@@ -1365,6 +1365,12 @@ def _(x):
 
 # List of available vocabularies
 ILS_VOCABULARIES = [
+    "acq_currency",
+    "acq_medium",
+    "acq_order_line_payment_mode",
```

Comment:
To keep it simple, I might be tempted to say `acq_order_line_payment_mode` = `acq_payment_mode`, so to keep only the latter, WDYT?

Reply:
@ntarocco I made it explicit because there is another [payment mode](https://github.com/inveniosoftware/invenio-app-ils/blob/4166c0bdc3bc58d904fd03079eeac529abe0754a/invenio_app_ils/acquisition/schemas/acq_orders/order-v1.0.0.json#L119)